### PR TITLE
github.io week 4

### PR DESCRIPTION
**Pull request by drumslayert**

### CODE REVIEW

**Comment by karjac14** [comment]
Great start! who doesn't like $ :+1:

**Comment by rosalsm** [comment]
Tools with Terry. Good idea :+1: 
Good Highlights for your journal too! :+1:

**Comment by ericquayson** [comment]
I like you're topic terry! :+1:

File: journal-week-4.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+###Journal week 4
+
+#####I. Hello `HTML`
+
+#####II. OMG!!! What the heck is this `$`?
+
+#####III. Shoot me now there's a chessboard!
+
+#####IV. It turns out I actually love `$`.
```
Comment (kellymurray) [inline journal-week-4.md:9]: yeah!! :dollar:

**Pull request merged**